src/components/MaterialUI/Navigation/TabBar.js

```diff
@@ -210,17 +227,20 @@ const MuiBar = ({ module, moduleName, pages }) => {
 				}}
 				ref={tabs}
 			>
-				{pages.map(({ href, label, outsideScope, close, path, params, mustTruncate }, index) => {
+				{pages.map(({ href, label, outsideScope, close, path, params, mustTruncate, icon, isDetails }, index) => {
 					let entityIdKey = Object.keys(params).find(p => p.toLowerCase().endsWith("id"));
 					if (!entityIdKey) entityIdKey = tryGetNewEntityIdKey(href);
 					const isModified = modifiedTabs.includes(href);
 					const isError = tabsWithErrors.includes(href);
 					const tabLabel = <TabLabel label={label} />;
+					const sectionIconClss = classNames(classes.moduleIcon, isDetails && classes.sectionIcon);
 					const tabClassName = classNames(
 						classes.labelContainer,
 						isModified && classes.modifiedLabel,
 						isError && classes.errorLabel,
 					);
+					const sectionIcon = !!icon && <Icon id={icon} className={sectionIconClss} />;
```

Comment:
Techniquement, seulement "icon" serait suffisant.... le double-bang n'est pas absolument nécessaire.

Reply:
going to remove the double negations before icon